lib/freedom_patches/ember_compat_handlebars.rb

```diff
@@ -1,32 +1,47 @@
 # barber patches to re-route raw compilation via ember compat handlebars
-#
 
-module Barber
-  class EmberCompatPrecompiler < Barber::Precompiler
```

Comment:
This is really annoying, we can no longer declare our own precompiler because `ember-handlebars-template` [assumes `Barber::Precompiler`](https://github.com/tricknotes/ember-handlebars-template/blob/263a74be2a7862455db34f0e9fdd2da1fbee1d91/lib/ember/handlebars/template.rb#L149) is the precompiler to be used.